src/components/Navbar/Searchbar/index.tsx

```diff
@@ -0,0 +1,310 @@
+				return result_image as string;
+			case 'project':
+				return '/images/assets/projects.svg';
+			case 'task':
+				return '/images/assets/tasks.svg';
+			default:
+				return '';
+		}
+	};
+
+	const ImageComponent = () => {
+		return (
+			<Image
+				src={getImageFromType(result_type)}
+				alt="search-result-img"
+				width={50}
+				height={50}
+				className="search-result-img"
+			/>
+		);
+	};
+
+	const BadgesComponent = () => {
+		const typeTooltip = `Type: ${result_type}`;
+		const orgTooltip = organization_name
+			? `Organization: ${organization_name}`
+			: '';
+		const projectTooltip = project_name ? `Project: ${project_name}` : '';
+
+		const OrganizationBadge = () => {
+			return (
+				<div
+					className="search-result-org-badge"
+					title={orgTooltip}
+				>
+					<svg
+						xmlns="http://www.w3.org/2000/svg"
+						fill="none"
+						viewBox="0 0 24 24"
+						strokeWidth={1.5}
+						stroke="currentColor"
+					>
+						<path
+							strokeLinecap="round"
+							strokeLinejoin="round"
+							d="M18 18.72a9.094 9.094 0 0 0 3.741-.479 3 3 0 0 0-4.682-2.72m.94 3.198.001.031c0 .225-.012.447-.037.666A11.944 11.944 0 0 1 12 21c-2.17 0-4.207-.576-5.963-1.584A6.062 6.062 0 0 1 6 18.719m12 0a5.971 5.971 0 0 0-.941-3.197m0 0A5.995 5.995 0 0 0 12 12.75a5.995 5.995 0 0 0-5.058 2.772m0 0a3 3 0 0 0-4.681 2.72 8.986 8.986 0 0 0 3.74.477m.94-3.197a5.971 5.971 0 0 0-.94 3.197M15 6.75a3 3 0 1 1-6 0 3 3 0 0 1 6 0Zm6 3a2.25 2.25 0 1 1-4.5 0 2.25 2.25 0 0 1 4.5 0Zm-13.5 0a2.25 2.25 0 1 1-4.5 0 2.25 2.25 0 0 1 4.5 0Z"
+						/>
+					</svg>
+					<div className="text-wrapper">
+						<span>{organization_name}</span>
+					</div>
+				</div>
+			);
+		};
+
+		const ProjectBadge = () => {
+			return (
+				<div
+					className="search-result-project-badge"
+					title={projectTooltip}
+				>
+					<svg
+						xmlns="http://www.w3.org/2000/svg"
+						fill="none"
+						viewBox="0 0 24 24"
+						strokeWidth={1.5}
+						stroke="currentColor"
+					>
+						<path
+							strokeLinecap="round"
+							strokeLinejoin="round"
+							d="M9 12.75 11.25 15 15 9.75M21 12a9 9 0 1 1-18 0 9 9 0 0 1 18 0Z"
+						/>
+					</svg>
+
+					<div className="text-wrapper">
+						<span>{project_name}</span>
+					</div>
+				</div>
+			);
+		};
+
+		const TypeBadge = () => {
+			return (
+				<div className="search-result-type-badge">{result_type}</div>
+			);
+		};
+
+		switch (result_type.toLowerCase()) {
+			case 'task':
+				return (
+					<div className="search-result-badges anim">
+						<TypeBadge />
+						<OrganizationBadge />
+						<ProjectBadge />
+					</div>
+				);
+			case 'project':
+				return (
+					<div className="search-result-badges anim">
+						<TypeBadge />
+						{<OrganizationBadge />}
+					</div>
+				);
+			case 'organization':
+			default:
+				return (
+					<div className="search-result-badges">
+						<TypeBadge />
+					</div>
+				);
+		}
+	};
+
+	const generateLink = (type: string, id: string) => {
```

Comment:
also will work nicely for notifications, again good work, but maybe add members ?

Reply:
Did not add members because not sure what association to give for users and projects, since our routing is setup as project/[id]/[user_id]